src/main/java/uk/gov/homeoffice/digital/sas/timecard/validators/timeentry/TimeEntryValidator.java

```diff
@@ -20,6 +20,45 @@ public class TimeEntryValidator implements ConstraintValidator<TimeEntryConstrai
   @Override
   public boolean isValid(TimeEntry timeEntry, ConstraintValidatorContext context) {
 
+    if (timeEntry.getActualEndTime() != null
+        && timeEntry.getActualStartTime().after(timeEntry.getActualEndTime())) {
+      String message = "End time must be after start time";
+      ClashingProperty clashingProperty = ClashingProperty.END_TIME;
```

Comment:
Is it an explicit requirement to categorise "start time after end time" error as a clash? One could argue that it's not really a clash, unless we call it an internal clash :)

Reply:
I didn't mean to remove `Clashing`  from the name of ClashingProperty, but rather pass a null clashingProperty value in this case, as it's not applicable, or maybe a new enum value `ClashingProperty.NONE` ?

Reply:
Got you. I think the naming ClashingProperty is what confused me, as this name implies this is purely to do with clashing time periods (ie. start time and end time fields only). Whereas in theory there could be another property that has a problem (ownerId as you suggested)
Maybe renaming `ClashingProperty` to `ProblematicProperty` or `ErroneousProperty` would make it less confusing? I feel `Property` is not descriptive enough
What do you think?
Feel free to pick a name more appropriate than the two I suggested above.

Reply:
I've gone with invalid field if you think that works?

Reply:
That works 👍